README.markdown

```diff
@@ -143,127 +137,176 @@ The following parameters are available in the ntp module:
 
 ####`autoupdate`
 
-**Deprecated:** This parameter determined whether the ntp module should be
-automatically updated to the latest version available.  Replaced by `package_ensure`.
+**Deprecated; replaced by the `package_ensure` parameter**
+Tells Puppet whether to keep the ntp module updated to the latest version available.
+Valid options: 'true' or 'false'
+Default value: 'false'
```

Comment:
are those supposed to be each on their own line in the rendered html?
cuz right now it ends up being on a single line.